pkg/security/controller/namespace_scc_allocation_controller.go

```diff
@@ -178,7 +182,21 @@ func (c *NamespaceSCCAllocationController) allocate(ns *corev1.Namespace) error
 		}
 	}
 
-	_, err = c.namespaceClient.Update(nsCopy)
+	encoder := legacyscheme.Codecs.LegacyCodec(legacyscheme.Scheme.PrioritizedVersionsAllGroups()...)
```

Comment:
install the core types into a scheme to do this.  You want to snip k/k links, not create them.